cocoa/src/toga_cocoa/widgets/numberinput.py

```diff
@@ -93,7 +93,7 @@ def create(self):
         # Add constraints to lay out the input and stepper.
         # Stepper is always top right corner.
         self.native.addConstraint(
-            NSLayoutConstraint.constraintWithItem_attribute_relatedBy_toItem_attribute_multiplier_constant_(
+            NSLayoutConstraint.constraintWithItem_attribute_relatedBy_toItem_attribute_multiplier_constant_(  # noqa: E501
```

Comment:
This was called out in the original ticket - Rubicon now has support for splitting these lines.